packages/labs/src/components/tag-input/_tag-input.scss

```diff
@@ -45,6 +45,11 @@ $ti-icon-padding-large: ($pt-input-height-large - $pt-icon-size-standard) / 2;
     padding: 0 ($input-padding-horizontal - $ti-padding);
     line-height: $tag-height;
 
+    // match padding of other input elements iff no tags are selected
+    &:first-child {
```

Comment:
That `:first-child` works really well in the context of tag input, but you probably wouldn't want this in other contexts. I'd be okay with changing the ghost input padding line 45 to what you have here, i.e. `padding: 0 $input-padding-horizontal;`. Then it's future proof for other components that'll need a ghost input (new input group layout with SVG icons for instance).

Then if we really want to polish `MultiSelect` and `TagInput` even more, we could do `&:not(:first-child) { padding: 0 ($input-padding-horizontal - $ti-padding) }` in the tag input CSS.

What do y'all think?